_infra/helm/collection-instrument/Chart.yaml

```diff
@@ -18,4 +18,4 @@ version: 1.1.4
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 2.0.57
+appVersion: 2.0.58
```

Comment:
This a fairly big change, so lets manually change this to 2.1.0

Reply:
ok

Reply:
agreed